data_dictionary/data_dictionary.cc

```diff
@@ -243,6 +261,50 @@ std::vector<view_ptr> keyspace_metadata::views() const {
             | boost::adaptors::transformed([] (auto&& s) { return view_ptr(s); }));
 }
 
+storage_options::value_type storage_options::from_map(std::string_view type, std::map<sstring, sstring> values) {
+    if (type == "NATIVE") {
```

Comment:
Let's have a descriptive name, like "LOCAL". Native works if you understand the project history, but coming from a clean slate you wouldn't guess what native means.

Also, let's reject such changes in ALTER (haven't checked if you do already).